program-runtime/src/execution_budget.rs

```diff
@@ -54,11 +64,29 @@ pub struct SVMTransactionExecutionBudget {
     pub heap_size: u32,
 }
 
+#[cfg(feature = "dev-context-only-utils")]
 impl Default for SVMTransactionExecutionBudget {
     fn default() -> Self {
         SVMTransactionExecutionBudget {
```

Comment:
Couldn't we call `SVMTransactionExecutionBudget::new_with_defaults(false)` here instead?